packages/suite-data/files/translations/es.json

```
@@ -961,7 +995,7 @@
   "TR_ONBOARDING_COINS_STEP": "Activar monedas",
   "TR_ONBOARDING_COINS_STEP_DESCRIPTION": "Selecciona las criptomonedas que se mostrarán en Trezor Suite. Puedes cambiar esta configuración en cualquier momento. Algunas monedas son tokens ERC20 y pueden utilizarse habilitando Ethereum más abajo.",
   "TR_ONBOARDING_CURRENT_VERSION": "Versión actual",
-  "TR_ONBOARDING_DATA_COLLECTION_DESCRIPTION": "Todos los datos son anónimos y se utilizan únicamente para el desarrollo de productos. Más información en los <a>Términos y condiciones</a>.",
+  "TR_ONBOARDING_DATA_COLLECTION_DESCRIPTION": "Todos los datos son anónimos y se utilizan únicamente con fines de desarrollo de productos. Lea más en nuestra documentación técnica <analytics></analytics> y <tos>Términos y condiciones</tos>.",
```

@alex-jerechinsky there should be something inside `<analytics></analytics>` used as a label for link
```suggestion
  "TR_ONBOARDING_DATA_COLLECTION_DESCRIPTION": "Todos los datos son anónimos y se utilizan únicamente con fines de desarrollo de productos. Lea más en nuestra <analytics>documentación técnica</analytics> y <tos>Términos y condiciones</tos>.",
```